src/main/java/edu/uc/inventorymanager/dto/ItemStatus.java

```diff
@@ -4,5 +4,5 @@
 
 public @Data class ItemStatus {
     private int id;
-    private String name;
+    private String ItemName;
```

Comment:
According to this article https://www.baeldung.com/java-clean-code ,it’s important for the variable name to be distinctive. The three dto classes had the same variable name for “name”. Even though they were in different classes, I think it’s still important for them to be different for future understanding.

Reply:
This is fine as it were.  "Item" is given, as it is in a class named ItemStatus.
Also, variable names should be lowerCamelCase, not UpperCamelCase.